library/ccm.c

```diff
@@ -363,31 +517,18 @@ int mbedtls_ccm_star_auth_decrypt( mbedtls_ccm_context *ctx, size_t length,
 {
```

Comment:
Why did you duplicate the code between `mbedtls_ccm_star_auth_decrypt` and `mbedtls_ccm_auth_decrypt`? They still do the same thing except for passing a different mode to `ccm_auth_crypt`.

Reply:
Passing different mode to ccm_auth_crypt is the only reason. I could add extra function including common code, parametrized with `mode` (basically same arguments as `ccm_auth_crypt`)